README.md

```diff
@@ -1,53 +1,70 @@
 Backdrop Contributed Project Group Application
 ==============================================
 
-Use this project to express interest in joining the growing community of developers maintaining a module or theme hosted at https://github.com/backdrop-contrib/.
+Interested in joining the growing Backdrop contributor community at https://github.com/backdrop-contrib?
 
 To apply simply create a request as an issue at https://github.com/backdrop-ops/contrib/issues/new
 
-You don't have to wait for access to Backdrop Contributed Project Group to share your code, nor is it required.  You can start writing or porting a project to Backdrop in your own GitHub account (or under a personal account hosted elsewhere). If you've already started a project, please include a link to the module or theme you have written. After joining the Backdrop Contributed Project Group, you will be able to transfer the entire repository to live under `https://github.com/backdrop-contrib/[your_project_name]`
+Please note: It is completely optional to post your modules, themes, or layouts in the Backdrop contrib group.
 
-Backdrop Contributed Project Author Agreement
----------------------------------------------
+Benefits of joining:
+- Benefit from official security procedures for keeping your code healthy.
+- Co-maintain your project with other Backdrop contrib developers.
+- Share the responsibilities of managing all the issue queues.
+- Collaborate with the larger Backdrop community.
 
-By publishing a project in the [Backdrop Contributed Project Group](https://github.com/backdrop-contrib) (Backdrop Contrib for short), you must agree to the following:
+Requirements to Join:
+- You must agree to the Backdrop Contributed Project Agreement, below.
+- You must submit your first Backdrop project for a quick spot-check / review.
+- If you are not submitting code for review at the time of application (for the
+  purposes of managing issues) you must have at least one recommendation from a
+  person who is already a member of the Backdrop contrib group.
```

Comment:
Still some line endings here. I think there must be a return before unordered lists to render correctly, so put a line after "Benefits of joining:" and "Requirements to Join:". Consistent capitalization between the two would be good also.